Main.py

```diff
@@ -102,6 +107,14 @@ def run():
     # \todo: make the code faster.
 
 
+
+def getGT(i,fc): # calculates the effective GT of agent i, fc should be FOUNDER_COUNT or SAMPLE_COUNT
```

Comment:
This should be possible to do for a number of indices in one go rather than just one.

You should probably send `sim_env` explicitly as the first argument (since it is by-ref).

Please make `i` and `fc` descriptive names.

Have a look at https://www.python.org/dev/peps/pep-0257/ and then there are some different styles of docstrings e.g. numpy-styled docstrings. Select one and use it. 

In general python doesn't like inline comments https://www.python.org/dev/peps/pep-0008/#inline-comments